### PR TITLE
Update localization.md

### LINKED ISSUE #14728: Point to Hisham's Blog until this is updated
http://hishambinateya.com/what-is-new-in-localization-in-asp.net-core-3.0

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d1833676-ff1b-c587-cc9c-81afffa8ef68
* Version Independent ID: ad989395-3f49-4d55-521e-cf6fe56281ee
* Content: [Globalization and localization in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/localization?view=aspnetcore-3.0#feedback)
* Content Source: [aspnetcore/fundamentals/localization.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/fundamentals/localization.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-fundamentals**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by Rick-Anderson**
@hishamco please review

Fixes  #14728

### CODE REVIEW

**Comment by hishamco** [comment]
I just plan to start with it, ok let me review it

**Comment by hishamco** [comment]
LGTM :shipit:

File: aspnetcore/fundamentals/localization.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ uid: fundamentals/localization
 
 By [Rick Anderson](https://twitter.com/RickAndMSFT), [Damien Bowden](https://twitter.com/damien_bod), [Bart Calixto](https://twitter.com/bartmax), [Nadeem Afana](https://afana.me/), and [Hisham Bin Ateya](https://twitter.com/hishambinateya)
 
+Until this document is updated for ASP.NET Core 3.0, see Hisham's Blog [What is new in Localization in ASP.NET Core 3.0](http://hishambinateya.com/what-is-new-in-localization-in-asp.net-core-3.0)
```
Comment (guardrex) [inline aspnetcore/fundamentals/localization.md]: Period at end ...

```suggestion
Until this document is updated for ASP.NET Core 3.0, see Hisham's Blog [What is new in Localization in ASP.NET Core 3.0](http://hishambinateya.com/what-is-new-in-localization-in-asp.net-core-3.0).
```

Comment (hishamco) [reply aspnetcore/fundamentals/localization.md]: Good catch

**Pull request merged**